OracleTuxedo/core/dockerfiles/12.2.2/Dockerfile

```diff
@@ -9,13 +9,13 @@
 #   sh build.sh
 
 # Pull base image
-FROM oracle/serverjre:8
+FROM container-registry.oracle.com/java/serverjre
 
 MAINTAINER Todd Little <todd.little@oracle.com>
```

Comment:
Is Todd Little still the maintainer? He's not the one submitting this update. If the maintainer has changed, please reflect that here.